api/client/swarm/leave.go

```diff
@@ -27,7 +27,7 @@ func newLeaveCommand(dockerCli *client.DockerCli) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.BoolVar(&opts.force, "force", false, "Force leave ignoring warnings.")
+	flags.BoolVar(&opts.force, "force", false, "Force a node to leave the swarm, ignoring warnings.")
```

Comment:
Maybe "Force this node...", since the command is always acting on the node where the command is run?